index.js

```diff
@@ -23,9 +23,8 @@ module.exports = whyIsNodeRunning
 function whyIsNodeRunning (logger) {
   if (!logger) logger = console
 
-  hook.disable()
   logger.error('There are %d handle(s) keeping the process running', active.size)
-  for (const o of active.values()) printStacks(o)
```

Comment:
why is this change needed?

Reply:
To be honest, I'm not really sure anymore. Given that `printStacks` is synchronous, I would assume it to make absolutely no difference because of the single-threaded nature, but I somehow distinctly remember that it did. I'm no longer working on the same project, so I unfortunately can't reproduce it.